tests/unit/blueprints/lib-test.js

```diff
@@ -5,7 +5,6 @@ const blueprintHelpers = require('ember-cli-blueprint-test-helpers/helpers');
 let setupTestHooks = blueprintHelpers.setupTestHooks;
```

Comment:
The `blueprint` blueprint doesn't contain the `.jshintrc` and `.eslintrc.js` files anymore, hence the removed code in this test.